src/interface/ui/base/List.hpp

```diff
@@ -94,7 +94,7 @@ class List : virtual public ExternalInterface,
     {
         return qt_model_;
     }
-    auto shutdown(std::promise<void>& promise) noexcept -> void
+    auto xshutdown(std::promise<void>& promise) noexcept -> void
```

Comment:
why this change ? to remove inheritance ? why 'x' ? does it mean sth or it is just random letter ?